docs/documentation/maintenance-coordinator.md

```diff
@@ -0,0 +1,91 @@
+
+<a name="change-management"></a>
+## Change Management
+Everybody may contribute to any aspect of the component. Such contributions 
+are handed in by pull requests or some other source of data if declared so in the components guidelines. 
+Pull requests on the public interface must be accepted by the JF. The coordinator gives a 
+recommendation to the JF on whether to accept or decline the PR. The decision of the JF may 
+be implicit if no objections to the recommendation of the coordinator is made. If no agreement 
+is achieved in the JF, the Technical Board will decide upon the request.
+
+Final implementations without further changes on the interface do not need 
+formal approval by the JF. The merge of the implementation is performed 
+by the coordinator or the coordinator may assign somebody to do so.
+
+Note that the general process for feature requests must be respected. However,
+this process is currently under review. The respective document will be linked as soon
+as available.
+
+<a name="issue-management"></a>
+## Issue Management
+Everybody is invited to make proposals on how to tackle any issue by proposing 
+a respective PR. Issues of the component must be reported as bugs. The coordinators 
+are responsible to assign the developer in charge on solving the bug. Due to the focus 
+on code quality, a low amount of bugs should be expected.
+
+<a name="scenarios"></a>
+## Scenarios
+This maintainance model is suited for components that have the potential 
+to grow too large to be handled by one single developer and therefore highly 
+benefit from contributions among different developers and even service providers. 
+It is further important that the component is of modular structure with different 
+parts following a similar scheme. It is especially suited, if some component is 
+of a critical importance for many other components, since it is designed to allow 
+a collaborative development of the vision for such a key aspect.
+
+<a name="expectations"></a>
+## What can be expected of a coordinator?
+* The coordinator MUST moderate the discussion on finding a vision on 
+the development of the component.
+* The coordinator MUST give recommendations to the JF whether to accept 
+or decline changes.
+* The coordinator MUST accept decisions of the Technical Board on change 
+requests in case of disagreement on the JF.
+* The coordinator MUST review final implementation of some accepted 
+interface change or organize some substitute to perform the review.
+* The coordinator MAY ask for funding to perform the review.
+* The coordinator MAY ask to split some interface change or implementation 
+into multiple pieces to make the change easier to understand.
+* The coordinator MUST ensure, that the documentation of the component is 
+up to date. The coordinator MAY ask to update the documentation as a condition 
+for accepting some change request.
+* The coordinator MUST ensure, that the automated unit tests are kept up 
```

Comment:
What about CI tools to make it easier for the coordinator to run the tests? Otherwise the coordinator needs the check out **every single PR** and execute the tests.

Reply:
I hope very very much, that the coordinator will use the CI for this (ci.ilias.de).

Reply:
@gvollbach any news on this? What was the reason we do not use those hooks again?

Reply:
The reason is that anyone could run code on our servers then. Like `exec("rm -rf /")`. We are currently looking into this.